var/spack/repos/builtin/packages/fpocket/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Fpocket(Package):
+    """fpocket is a very fast open source protein pocket detection algorithm
+       based on Voronoi tessellation."""
+
+    homepage = "https://github.com/Discngine/fpocket"
+    version('develop', branch='master',
```

Comment:
You can actually call this version `master` now that #1983 has been merged.

Reply:
done